src/client.ts

```diff
@@ -1863,7 +1863,7 @@ export class StreamChat<StreamChatGenerics extends ExtendableGenerics = DefaultG
       mark_messages_deleted?: boolean;
     },
   ) {
-    return await this.delete<APIResponse & { user: UserResponse<StreamChatGenerics> }>(
+    return await this.delete<APIResponse & { user: UserResponse<StreamChatGenerics> } & { task_id: string }>(
```

Comment:
good catch but it should be optional because it's only returned server side where we run tests. Client side it won't be returned

Reply:
```suggestion
    return await this.delete<APIResponse & { user: UserResponse<StreamChatGenerics> } & { task_id?: string }>(
```